heracles/result.py

```diff
@@ -88,6 +88,68 @@ def __array_wrap__(self, arr, context=None, return_scalar=False):
         return out.view(np.ndarray)
 
 
+class CovMatrix(np.ndarray):
+    """
+    NumPy :class:`~numpy.ndarray` subclass with extra properties for
+    two-point covariance matrix.
+    """
+
+    __slots__ = (
+        "axis",
```

Comment:
Let's completely remove `.axis` from these.

If the object is a `CovMatrix`, the axes are `(-2, -1)`. So we have that info encoded in the type of the object. No need to carry it around.

If it became necessary, we could have a `result_ell_axis()` function that returns the correct axis for each type of result.

Reply:
@ntessore  sorted! Please press the green button if you are happy.